benchmarks/free_surface_tractions/viscous/free_surface_viscous_cylinder_2D_loading.prm

```diff
@@ -0,0 +1,159 @@
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = single Advection, single Stokes
+set CFL number                             = 0.5
+set Maximum time step                      = 10
+set Output directory                       = output_free_surface_viscous_cylinder_2D_loading
+set Timing output frequency                = 1
+set Pressure normalization                 = no
+
+# Model geometry (500x500 km, 50 km spacing)
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X repetitions = 10
+    set Y repetitions = 10
+    set X extent      = 500e3
+    set Y extent      = 500e3
+  end
+end
+
+# Mesh refinement specifications 
+subsection Mesh refinement
+  set Initial adaptive refinement        = 1
+  set Initial global refinement          = 1
+  set Time steps between mesh refinement = 1
+  set Strategy				 = strain rate
+end
+
+# Element types
+subsection Discretization
+  set Composition polynomial degree     = 2
+  set Stokes velocity polynomial degree = 2
+  set Temperature polynomial degree     = 1
+end
+
+subsection Free surface
+  set Free surface boundary indicators = top
+  set Surface velocity projection      = normal
+  set Additional tangential mesh velocity boundary indicators = left,right
+end
+
+# Velocity boundary conditions
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = bottom, left, right 
+end
+
+# Prescribe a fixed vertical traction on the top boundary
+subsection Boundary traction model
+  set Prescribed traction boundary indicators = top y: function
+end
+
+subsection Boundary traction model
+  subsection Function
+    set Variable names = x,y,t
+    set Function constants  = r0=100.e3, H0=1.e3, t1=1.e3, rhoi=900, g=9.8, t0=1.e3
+        # r0 is load radius, H0 is load height, t1 is time load is removed, 
+	# rhoi is density of ice/load, g is gravitational acceleration 
+    set Function expression = 0; if (x<r0 && t<t1,-g*H0*rhoi*(1-t/t1), 0)
+    #set Function expression = 0; if (x<r0 ,if(t<t0,-g*H0*rhoi,if(t<t1,-g*H0*rhoi*(1-(t-t0)/t1),0)), 0)
+  end
+end
+
+# Number and name of compositional fields
+subsection Compositional fields
+  set Number of fields = 3
+  set Names of fields  = stress_xx, stress_yy, stress_xy
+end
+
+# Spatial domain of different compositional fields
+subsection Initial composition model
+  set Model name = function
+  subsection Function
+    set Variable names      = x,y
+    set Function constants  = 
+    set Function expression = 0; 0; 0;
+  end
+end
+
+# Composition boundary conditions
+subsection Boundary composition model
+  set Allow fixed composition on outflow boundaries = true
+  set Fixed composition boundary indicators   = 
+  set List of model names = initial composition
+end
+
+# Temperature boundary conditions
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators = bottom, top, left, right
+  set List of model names = box
```

Comment:
You can change this line to ` set List of model names = initial temperature`, which would allow deleting the `Box` subsection below.